main/lispBM/src/lbm_memory.c

```diff
@@ -307,7 +307,7 @@ static lbm_uint *lbm_memory_allocate_internal(lbm_uint num_words) {
 }
 
 lbm_uint *lbm_memory_allocate(lbm_uint num_words) {
-  if (memory_num_free - num_words < memory_reserve_level) {
+    if (memory_num_free - num_words < memory_reserve_level) {
```

Comment:
Is this a mistake? :)

Reply:
Oh no, that looks like one! Will fix once I'm back at my computer

Reply:
Force pushed a fix!